src/@next/Currency/Currency.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { InputProps } from '../Input/Input';
+import { StyledCurrency } from './CurrencyStyles';
+
+export type CurrencyProps = Omit<
+  InputProps,
+  'type' | 'prefix' | 'onChange' | 'value'
+> & {
+  locale?: string;
+  value?: number;
+  onChange?: (value: number) => void;
+};
+
+export const Currency = ({
```

Comment:
should we add `*Input` suffix to them to clearly indicate these component are Input components?

Reply:
updated